errbot/backends/xmpp.py

```diff
@@ -368,6 +372,7 @@ def __init__(self, config):
 
         self.jid = Identifier(identity['username'])
         self.password = identity['password']
+        self.server = identity.get(['server'], None)
```

Comment:
I overlooked this when reviewing & merging :( This should be `self.server = identity.get('server', None)` instead. Going to push a fix to master.